homeassistant/components/lutron/fan.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+from typing import Any
+import logging
+
+from homeassistant.components.fan import FanEntity, FanEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+
+from . import LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+FULL_SUPPORT = (
+    FanEntityFeature.SET_SPEED
+)
+
+async def async_setup_platform(

```

Comment:
Remove `async_setup_platform()` and implement the setup from `async_setup_entry()`

Reply:
this is a minor enhancement to the lutron integration.  to keep this entity consistent with the other entities I changed the implementation to setup_platform()